static/js/load_schedule.js

```diff
@@ -1,75 +1,50 @@
-                $('div.title2').css('display','none');
-                $('img#pic').css('display','none');
-                $('div.like_count').css('display','none');
-                $('img#event_qrcode').css('display','none');
-                $('div#user_pref').css('display','none');
-                $('div#news').css('display','none');
-                $('img#event_qrcode').css('display','none');
-                $('img#qrcode1').css('display','none');
-                $('img#qrcode2').css('display','none');
-                
                 console.log("text");
+                last_schedule_id = jsonRes.schedule_id;
+                resetTextDisplay();
+                // 1. preference //
                 if ('preference' in jsonRes.file_text){
-                    console.log(jsonRes);
-                    last_schedule_id = jsonRes.schedule_id;
-                    $('div#user_pref').css('display','inline');
-                    $('img#qrcode1').css('display','inline');
-                    $('img#qrcode2').css('display','inline');
+                    setPreferenceDisplay();
                     if(jsonRes.file_text.preference === 1){
-                        var weekday=new Array(7);
-                        weekday[0]="Mon.";
-                        weekday[1]="Tue.";
-                        weekday[2]="Wed.";
-                        weekday[3]="Thu.";
-                        weekday[4]="Fri.";
-                        weekday[5]="Sat.";
-                        weekday[6]="Sun.";
-                        var datetime = new Date(Date.parse(jsonRes.file_text.date));
-                        console.log(datetime);
-                        var weekday = weekday[datetime.getDay()];
-                        var year = datetime.getFullYear();
-                        var month = datetime.getMonth() + 1;
-                        var date = datetime.getDate();
-                        console.log("Month",month);
-                        console.log("Year",year);
-                        console.log("Date",date);
-                        $('div#week b').text(weekday);
-                        $('div#year b').text(year);
-                        $('div#date b').text(String(month)+'.'+String(date));
-                        $('div#name b').text(jsonRes.file_text.nickname);
-                        $('div#star b').text(jsonRes.file_text.constellation.name);
-                        $('div#overall b').text(jsonRes.file_text.constellation.value[0]);
-                        $('div#love b').text(jsonRes.file_text.constellation.value[1]);
-                        $('div#career b').text(jsonRes.file_text.constellation.value[2]);
-                        $('div#wealth b').text(jsonRes.file_text.constellation.value[3]);
+                        setDateInfo();
+                        setConstellationInfo(jsonRes);
                         if( jsonRes.file_text.news.length > 0){
                             console.log("print!!!");
```

Comment:
Please help us, remove `console.log("print!!!");` in line 49

Reply:
Actually, I wanna replace the code from line 49 to line 52 with function.
But I don't know how to name the function differ from the forum news below.